web/tuning/common.ts

```diff
@@ -78,6 +78,12 @@ function fixVels(ts: number[], xs: number[], vs: number[]) {
   if (ts.length !== xs.length || ts.length !== vs.length) {
     throw new Error(`${ts.length} !== ${xs.length} !== ${vs.length}`);
   }
+  // check if the second to last number in the data is non-integer
+  if (xs[-2] !== Math.round(xs[-2])) {
+    return numDerivOffline(ts, xs).map((est, i) => {
+      return (vs[i + 1], est)
```

Comment:
What do you intend for this to do? I'm assuming you don't want the comma operator here (or maybe I'm missing something).

And doesn't this conflate "double positions" with "missing velocities"?

Reply:
My only goal with this change is to not run inverseOverflow on the data if it isn't an integer. I am honestly uncertain that this is a great way of detecting and doing that, and very open to feedback on better ways.